pages/glossary/block-of-content.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Block of content
+key: block-of-content
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - CSS styling
+  - DOM tree
+---
+
+A _block of content_ in an [HTML web page][] is a set of nodes from that page such that:
```

Comment:
Discussion at https://github.com/act-rules/act-rules.github.io/pull/421#discussion_r496643177 + double checking with a native English speaker (content writer at Siteimprove), conclusion is that "an HTML" is correct because it is pronounced "an aitch-tee-em-el".

Reply:
Well explained @Jym77 💯